src/opencl/cl_accelerator.cpp

```diff
@@ -141,6 +142,15 @@ namespace spla {
             // Likely, it is an integrated amd device
             if (m_max_wgs <= 256 || m_max_cu == 1) m_wave_size = 16;
         }
+        if (m_vendor_name.find("Imagination Technologies") != std::string::npos ||
+            m_vendor_name.find("IMG") != std::string::npos ||
+            m_vendor_name.find("img") != std::string::npos ||
+            m_vendor_id == 0x1010) {
+            m_vendor_code = VENDOR_CODE_IMG;
+            m_default_wgs = 32;
```

Comment:
Why you decided to set default wgs = 32, and not 64? It seems that 64 suits in most cases well

Reply:
> RECOMMENDATION: If possible use a workgroup size of 32.

This is a recommendation from the [official Imagination Technologies blog](https://blog.imaginationtech.com/a-quick-guide-to-writing-opencl-kernels-for-rogue/)

Reply:
Done